### PR TITLE
Update README.md for additionnal computer resources (node006/007)

### PR DESCRIPTION
**Pull request by 4rnaudB**
Adding information for node006 and node007 with GPU specs.

### CODE REVIEW

**Comment by 4rnaudB** [comment]
Do we want to add a specific use for these computers? (ex: Peterson is indicated as: CST simulations)

**Comment by jcohenadad** [comment]
> Do we want to add a specific use for these computers? (ex: Peterson is indicated as: CST simulations)

I don't think so, but we should at least mention that CST is installed in them.
Also, mention the OS (it is windows I suppose?)

**Comment by 4rnaudB** [comment]
Yes, I wrote Windows 10 in OS and added that CST is installed.

File: computing-resources/neuropoly/README.md [hunk 1]
```diff
@@ -457,6 +457,24 @@ This computer is to be used for programming pulse sequences within the Siemens I
 
 This computer is to be used for CST simulations. The station is bound to the **Poly-Grames** domain.
 
+### node006
```
Comment (jcohenadad) [inline computing-resources/neuropoly/README.md:460]:
```suggestion
### node006 (Poly-Grames)
```

File: computing-resources/neuropoly/README.md [hunk 2]
```diff
@@ -457,6 +457,24 @@ This computer is to be used for programming pulse sequences within the Siemens I
...
+| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Model**       | PC                                                                                                                                                                      |
+| **OS**          | Windows 10                                                                                                                                                              |
+| **Hostname**    | `node006.grames.polymtl.ca`                                                                                                                                            |
+| **Credentials** | [NeuroPoly Internal Document: Peterson Remote Credentials](https://docs.google.com/document/d/13iNhiBKYZWT9ytsvYeeYV4FJn6Wn00q9Ctka7toMV08/edit#heading=h.3ssvul9a335m) |
+| **GPU**       | NVIDIA Tesla V100S-PCIE-32GB                                                                                                                                                           |
+### node007
```
Comment (jcohenadad) [inline computing-resources/neuropoly/README.md:472]:
```suggestion
### node007 (Poly-Grames)
```

File: computing-resources/neuropoly/README.md [hunk 3]
```diff
@@ -457,6 +457,29 @@ This computer is to be used for programming pulse sequences within the Siemens I
...
+
+### node007 (Poly-Grames)
+
+| Spec            | Description                                                                                                                                                             |
+| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Model**       | PC                                                                                                                                                                      |
+| **OS**          | Windows 10                                                                                                                                                              |
+| **Hostname**    | `node007.grames.polymtl.ca`                                                                                                                                            |
+| **Credentials** | [NeuroPoly Internal Document: Peterson Remote Credentials](https://docs.google.com/document/d/13iNhiBKYZWT9ytsvYeeYV4FJn6Wn00q9Ctka7toMV08/edit#heading=h.ovkq2tj4p4hz) |
```
Comment (jcohenadad) [inline computing-resources/neuropoly/README.md:479]:
```suggestion
| **Credentials** | [NeuroPoly Internal Document: node007 Remote Credentials](https://docs.google.com/document/d/13iNhiBKYZWT9ytsvYeeYV4FJn6Wn00q9Ctka7toMV08/edit#heading=h.ovkq2tj4p4hz) |
```

**Comment by 4rnaudB** [comment]
Good for merge?

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**